src/router.js

```diff
@@ -55,6 +56,19 @@ const router = new Router({
       path: '/users/add',
       name: 'users-add',
       component: () => import(/* webpackChunkName: "about" */ './views/admin/UserAdd.vue')
+    },
+    {
+      path: '/projects',
+      name: 'project_list',
+      component: List,
+      meta: {
+        requiresAuth: true
+      }
+    },
+    {
+      path: '/projects/add',
+      name: 'projects-add',
```

Comment:
change the name into `project_add` - we should follow a single pattern for routing names, you have `project_list` and `projects-add`, first of all, the prefix should be the same, either `project_` or `projects_`, I would chose singular version - `project_`, and use `_` as a separator instead of `-`.